dis_snek/api/http/http_requests/channels.py

```diff
@@ -72,7 +72,7 @@ async def create_guild_channel(
         name: str,
         channel_type: Union["ChannelTypes", int],
         topic: Absent[Optional[str]] = MISSING,
-        position: int = 0,
+        position: Optional[int] = None,
```

Comment:
Absent[int] = MISSING

Reply:
Are you sure?

Reply:
This doesn't need to be missing, surely

Reply:
We really need to nack down which to use `Absent` or `Optional` in context like this.